client/src/pages/ProfileDetails/BookingForm/useStyles.ts

```diff
@@ -0,0 +1,72 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  bookingFormCard: {
+    width: '80%',
+    minWidth: 400,
+    height: '55%',
+    minHeight: 550,
+    justifySelf: 'flex-end',
+  },
+  bookingForm: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  userRateText: {
+    marginTop: 40,
+    marginBottom: 10,
+    color: 'black',
+    fontSize: 22,
+    fontWeight: 'bold',
+  },
+  profileDetailsForm: {
+    width: '100%',
+    marginTop: 50,
+    display: 'flex',
```

Comment:
Most of these styles can be added directly to the Box component above

Reply:
I was getting some spacing issues when adding directly, so I kept them in the useStyles file